supervisor/install/install_darwin.go

```diff
@@ -42,10 +43,6 @@ const plistTpl = `
 	</array>
 	<key>KeepAlive</key>
 	<true/>
-	<key>StandardOutPath</key>
-	<string>{{.LogPath}}</string>
```

Comment:
Do we really want to remove it?
I guess this logs more, like fatal fails, failed startups or crashes. Or just standard and error outputs?

Reply:
From the documentation it says that this is only for application logs. System logs are stored in different directions.